scripts/src/admin/translations.ts

```diff
@@ -100,6 +105,7 @@ function constructRowsFromArray(keyBase: string, arr: any[], rowConstructor: Row
 
 function constructRowsFromObj(keyBase: string, obj: { [key: string]: any }, rowConstructor: RowConstructor) {
 	if (!obj) {
```

Comment:
how can `!obj` be false? the function definition doesn't show this is a possibility